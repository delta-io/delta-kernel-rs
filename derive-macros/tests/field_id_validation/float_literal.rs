use delta_kernel_derive::ToSchema;

#[derive(ToSchema)]
struct TestStruct {
    #[field_id = 123.45]  // Should be integer, not float
    field: String,
}

fn main() {}
