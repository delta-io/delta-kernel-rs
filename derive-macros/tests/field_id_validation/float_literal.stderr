error: field_id error: Expected field-id to be an integer
 --> tests/field_id_validation/float_literal.rs:5:18
  |
5 |     #[field_id = 123.45]  // Should be integer, not float
  |                  ^^^^^^
